lib/foreman_theme_satellite/engine.rb

```diff
@@ -121,11 +121,11 @@ def self.metadata_field(key, default)
     metadata_path = Rails.env.production? ? METADATA_PATH : "#{__dir__}/../../config/metadata.yml"
 
     @metadata_yaml ||= File.exist?(metadata_path) ? YAML.load_file(metadata_path) : {}
-    @metadata_yaml[key] || default
+    @metadata_yaml.fetch(key, default)
   end
 
   def self.get_satellite_version
-    metadata_field('version', '0.0.0-development')
+    metadata_field('version', '6.13.0-development')
```

Comment:
I thought about using the gem version, but this gem is already at 13.1.0 so I think this is cleaner. Should we bump this to 6.15 while we're at it?

Reply:
:woman_shrugging: